faker/providers/python/__init__.py

```diff
@@ -151,10 +151,59 @@ def pyint(self, min_value=0, max_value=9999, step=1):
 
     def pydecimal(self, left_digits=None, right_digits=None, positive=False,
                   min_value=None, max_value=None):
+        if left_digits is not None and left_digits < 0:
+            raise ValueError(
+                'A decimal number cannot have less than 0 digits in its '
+                'integer part')
+        if right_digits is not None and right_digits < 0:
+            raise ValueError(
+                'A decimal number cannot have less than 0 digits in its '
+                'fractional part')
+        if (not left_digits is None and left_digits == 0) and (not right_digits is None and right_digits == 0):
```

Comment:
`faker/providers/python/__init__.py:162:13: E714 test for object identity should be 'is not'`

```suggestion
        if (left_digits is not None and left_digits == 0) and (right_digits is not None and right_digits == 0):
```